contexts/ProposalMarketsContext.tsx

```diff
@@ -652,6 +661,8 @@ export function ProposalMarketsProvider({
   );
 
   return (
-    <ProposalMarketsContext.Provider value={memoValue}>{children}</ProposalMarketsContext.Provider>
+    <ProposalMarketsContext.Provider value={memoValue}>
+      <BalancesProvider>{children}</BalancesProvider>
```

Comment:
@R-K-H I had to add this here so that the balances could pick up the update from the query cache... there are better ways to do this but again, if we are only in this codebase for one more week, just something to consider.